java/arcs/core/entity/EntityBase.kt

```diff
@@ -230,6 +231,13 @@ open class EntityBase(
         }
         value.forEach { checkType(field, it, type.primitiveType, "member of ") }
       }
+      is FieldType.NullableOf -> {
+        // if (value == null) {
```

Comment:
It's generally preferable not to comment out code, and instead to just have a descriptive TODO that says what you want to do. Commented-out code has a habit of becoming out of date...

Reply:
Happy to just leave a TODO, the code was only left as a reminder of my intent.